src/cpdb/components/geometry_visualization_report.py

```diff
@@ -0,0 +1,20 @@
+import streamlit as st  # type: ignore
+from src.constants import COLOR_SCHEME
+
+def geometry_visualization_report(data: dict):
+
+    st.header(
+        f"Visualize Geometries"
+    )
+
+    st.markdown(
+        f"""
+        To quickly assess whether the geometries from the shapefiles are corrupted or not. 
```

Comment:
Also probably overkill but description could be better "The the idea behind these maps is to guide engineers in figuring out if the shapefiles are corrupted but also (in conjunction with the table) which source spatial data files might be causing the issue (which historically has been an issue in the data library upload specifically with the source to destination projection). If records are falling outside of NYC from a specific agency, that might indicate an issue upstream with how the data is being uploaded into data library"